src/test/resources/cxf/Recurring.wsdl

```diff
@@ -0,0 +1,604 @@
+          <xsd:element minOccurs="0" name="lastName" nillable="true" type="xsd:string" />
+        </xsd:sequence>
+      </xsd:complexType>
+      <xsd:simpleType name="Gender">
+        <xsd:restriction base="xsd:string">
+          <xsd:enumeration value="MALE" />
+          <xsd:enumeration value="FEMALE" />
+          <xsd:enumeration value="UNKNOWN" />
+        </xsd:restriction>
+      </xsd:simpleType>
+      <xsd:complexType name="ServiceException">
+        <xsd:sequence>
+          <xsd:element minOccurs="0" name="error" nillable="true" type="ns1:Error" />
+          <xsd:element minOccurs="0" name="type" nillable="true" type="ns1:Type" />
+        </xsd:sequence>
+      </xsd:complexType>
+      <xsd:simpleType name="Error">
+        <xsd:restriction base="xsd:string">
+          <xsd:enumeration value="Unknown" />
+          <xsd:enumeration value="NotAllowed" />
+          <xsd:enumeration value="NoAmountSpecified" />
+          <xsd:enumeration value="UnableToDetermineVariant" />
+          <xsd:enumeration value="InvalidMerchantAccount" />
+          <xsd:enumeration value="RequestMissing" />
+          <xsd:enumeration value="InternalError" />
+          <xsd:enumeration value="UnableToProcess" />
+          <xsd:enumeration value="PaymentDetailsAreNotSupported" />
+        </xsd:restriction>
+      </xsd:simpleType>
+      <xsd:simpleType name="Type">
+        <xsd:restriction base="xsd:string">
+          <xsd:enumeration value="internal" />
+          <xsd:enumeration value="validation" />
+          <xsd:enumeration value="security" />
+          <xsd:enumeration value="configuration" />
+        </xsd:restriction>
+      </xsd:simpleType>
+    </xsd:schema>
+    <xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema" attributeFormDefault="qualified" elementFormDefault="qualified" targetNamespace="http://payment.services.adyen.com">
+      <xsd:import namespace="http://common.services.adyen.com" />
+      <xsd:complexType name="BankAccount">
+        <xsd:sequence>
+          <xsd:element minOccurs="0" name="bankAccountNumber" nillable="true" type="xsd:string" />
+          <xsd:element minOccurs="0" name="bankLocationId" nillable="true" type="xsd:string" />
+          <xsd:element minOccurs="0" name="bankName" nillable="true" type="xsd:string" />
+          <xsd:element minOccurs="0" name="bic" nillable="true" type="xsd:string" />
+          <xsd:element minOccurs="0" name="countryCode" nillable="true" type="xsd:string" />
+          <xsd:element minOccurs="0" name="iban" nillable="true" type="xsd:string" />
+          <xsd:element minOccurs="0" name="ownerName" nillable="true" type="xsd:string" />
+        </xsd:sequence>
+      </xsd:complexType>
+      <xsd:complexType name="Card">
+        <xsd:sequence>
+          <xsd:element minOccurs="0" name="billingAddress" nillable="true" type="ns1:Address" />
+          <xsd:element minOccurs="0" name="cvc" nillable="true">
```

Comment:
as discussed, could you add

          <xsd:element minOccurs="0" name="brand" nillable="true" type="xsd:string"/>

here? To be independent from class loader ordering.